SpatialGDK/Source/SpatialGDK/Public/Interop/OwnershipCompletenessHandler.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "Misc/Optional.h"
+
+#include "SpatialCommonTypes.h"
+#include "SpatialView/SubView.h"
+
+class USpatialNetDriver;
+
+namespace SpatialGDK
+{
+class ViewCoordinator;
+
+class FOwnershipCompletenessHandler
+{
+public:
+	FOwnershipCompletenessHandler(bool bInIsServer)
+		: bIsServer(bInIsServer)
+	{
+	}
+	bool IsOwnershipComplete(Worker_EntityId EntityId, const EntityViewElement& Entity) const;
+	void AddPlayerEntity(Worker_EntityId EntityId);
+	void RemovePlayerEntity(Worker_EntityId EntityId);
+	void AddSubView(FSubView& InSubView);
+
+	static TArray<FDispatcherRefreshCallback> GetCallbacks(ViewCoordinator& Coordinator);
+
+private:
+	bool bIsServer;
```

Comment:
nit - no value at this time, but could be a 'uint32 bIsServer : 1' in the future.

Reply:
I don't think there's a lot to save there since we only create a single instance of this class per a NetDriver.